### DISCUSSION TITLE
Laragon for macbook

### DISCUSSION THREAD

**Discussion by Chris48-sys** [leokhoa/laragon#857] [General]
hello friends, 
who can help me please? i'm looking for a version of laragon for macbook

**Comment by Diego-Franke** [comment]
Just toss that overpriced piece of junk in the trash. MacBooks are useless for real development. Get a real machine.

**Comment by Pen-y-Fan** [comment]
I've used a Macbook for over a year, I would recommend [Valet](https://laravel.com/docs/11.x/valet) and [PHP Monitor](https://phpmon.app/). They are both free to use and are close to Laragon's functionality. Any additional software can be installed using the Mac package manager [brew](https://brew.sh/).

An alternative recently released is [herd](https://herd.laravel.com/docs/1/getting-started/about-herd), it uses a self-contained PHP binary and valet. This makes installing and switching PHP versions much easier. I haven't used it, it is free with additional pro-features are paid.

I hope you enjoy your Mac, if found mine to be a blisteringly fast and productive development environment.

**Reply by Chris48-sys** [reply]
thank you for your informative answer, I'll give it a try.

**Reply by Pen-y-Fan** [reply]
I've created a GitHub gist of my notes for my Mac setup, feel free to pick and choose what you need. The notes are very basic; I hope you find it useful.

[Setup Mac for Development](https://gist.github.com/Pen-y-Fan/7ecb3ed19c417d94d17e93582622178c)

**Reply by Chris48-sys** [reply]
Hello, 
Thank you for your availability and your help.
I really appreciate it.

**Comment by christianpasinrey** [comment]
MAMP
